tests/qunit/setup-editor.js

```diff
@@ -153,16 +204,8 @@ Marionette.Region.prototype.attachHtml = () => {
 
 Marionette.CompositeView.prototype.getChildViewContainer = ( containerView ) => {
 	containerView.$childViewContainer = jQuery( '<div />' );
-	containerView.$childViewContainer.appendTo( '#qunit-fixture' );
+	containerView.$childViewContainer.appendTo(
+		jQuery( '#elementor-preview-iframe' ).contents().find( '#elementor' )
+	);
 	return containerView.$childViewContainer;
 };
-
-fixture.setBase( 'tests/qunit' );
-
-QUnit.testStart( ( { module, name } ) => {
-	fixture.load( 'index.html' );
-} );
-
-QUnit.testDone( ( { module, name } ) => {
-	fixture.cleanup();
```

Comment:
Why?

Reply:
I added this manually without using fixture.